test/v4/amendHistoricNonFhlUkPropertyPeriodSummary/AmendHistoricNonFhlUkPropertyPeriodSummaryConnectorSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v4.amendHistoricNonFhlUkPropertyPeriodSummary
+
+import api.connectors.{ConnectorSpec, DownstreamOutcome}
+import api.models.domain.{Nino, PeriodId}
+import api.models.outcomes.ResponseWrapper
+import org.scalamock.handlers.CallHandler
+import v4.amendHistoricNonFhlUkPropertyPeriodSummary.model.request.{
+  AmendHistoricNonFhlUkPropertyPeriodSummaryRequestData,
+  Def1_AmendHistoricNonFhlUkPropertyPeriodSummaryRequestBody,
+  Def1_AmendHistoricNonFhlUkPropertyPeriodSummaryRequestData
+}
+import v4.amendHistoricNonFhlUkPropertyPeriodSummary.model.response.AmendHistoricNonFhlUkPropertyPeriodSummaryResponse
+
+import scala.concurrent.Future
+
+class AmendHistoricNonFhlUkPropertyPeriodSummaryConnectorSpec extends ConnectorSpec {
+
+  private val nino     = Nino("AA123456A")
+  private val periodId = PeriodId(from = "2017-04-06", to = "2017-07-04")
+
+  "The connector" when {
+    "sending a valid amend request" should {
+      "return the ok result" in new IfsTest with Test {
+        val response: AmendHistoricNonFhlUkPropertyPeriodSummaryResponse =
+          AmendHistoricNonFhlUkPropertyPeriodSummaryResponse(transactionReference = "2017090920170909")
+
+        val outcome: Right[Nothing, ResponseWrapper[AmendHistoricNonFhlUkPropertyPeriodSummaryResponse]] = Right(ResponseWrapper(correlationId, response))
+
+        stubHttpResponse(outcome)
+
+        val result: DownstreamOutcome[Unit] = await(connector.amend(request))
+        result shouldBe outcome
+      }
+    }
+  }
+
+  trait Test {
+    _: ConnectorTest =>
+
+    protected val connector: AmendHistoricNonFhlUkPropertyPeriodSummaryConnector = new AmendHistoricNonFhlUkPropertyPeriodSummaryConnector(
+      http = mockHttpClient,
+      appConfig = mockAppConfig
+    )
+
+    private val requestBody: Def1_AmendHistoricNonFhlUkPropertyPeriodSummaryRequestBody =
+      Def1_AmendHistoricNonFhlUkPropertyPeriodSummaryRequestBody(None, None)
+
+    protected val request: AmendHistoricNonFhlUkPropertyPeriodSummaryRequestData =
+      Def1_AmendHistoricNonFhlUkPropertyPeriodSummaryRequestData(nino, periodId, requestBody)
+
+    def pathFrom(request: AmendHistoricNonFhlUkPropertyPeriodSummaryRequestData): String =
+      s"income-tax/nino/${request.nino.value}/uk-properties/other/periodic-summaries" +
```

Comment:
```suggestion
s"income-tax/nino/${request.nino.value}/uk-properties/other/periodic-summaries?from=${request.periodId.from}&to=${request.periodId.to}"

Reply:
It's how it was before; I think it's fine as-is, makes the code clearer